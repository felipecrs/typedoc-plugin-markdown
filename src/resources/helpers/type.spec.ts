import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] type`, () => {
  let project: any;

  beforeAll(() => {
    ({ project } = setup());
  });

  test(`should compile intrinsic type`, () => {
    const data = project.findReflectionByName('color');
    const result = Handlebars.helpers.type.call(data.type);
    expect(result).toMatchSnapshot();
  });
});
