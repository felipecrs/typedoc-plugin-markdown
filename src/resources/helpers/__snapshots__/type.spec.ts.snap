// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] type should compile intrinsic type 1`] = `"string"`;
